packages/manager/modules/dbaas-logs/src/detail/options/addtool/index.js

```diff
@@ -0,0 +1,29 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-api-services';
+import 'ovh-ui-angular';
+
+import 'ovh-ui-kit/dist/oui.css';
```

Comment:
look like oui css is imported multiple times, 28 times, can you import it once in root file?

Reply:
The same as bootstrap... Since the module has to be independent, I have the import in each module. Please let me know what you think everyone :)

Reply:
it seems ravi is asking to import only it once at the root of dbaas-logs modules, is that right ? i think it make sense since the module is the smallest part that we can reuse (import) elsewhere.

Reply:
Agreed. This is done :)